versioneer.py

```diff
@@ -0,0 +1,1822 @@
+
+# Version: 0.18
```

Comment:
will we ever need to upgrade this?))

Reply:
Yes its documented how to upgrade here https://github.com/warner/python-versioneer#updating-versioneer 

I will add a link

Reply:
It's already documented in the doc string below. I think this is ok

Reply:
yes, it is perfect! thanks for pointing me there